invoices/tenkfeet_api.py

```diff
@@ -2,205 +2,222 @@
 
 import requests
 import schema
-from django.utils import timezone
-from django.utils.dateparse import parse_date, parse_datetime
+from schema import And, Or, Schema, Use
 
+from invoices.utils import parse_date, parse_datetime, parse_float
+
+
+def opt(t):
+    return Or(type(None), t)
+
+
+def zero_to_none(a):
```

Comment:
Is this used somewhere?

Reply:
That seems to be left over from using second edition of hidden API, which in end didn't have required info. So no.